test/fixtures/bpmn/model-complete-semantic.bpmn

```diff
@@ -772,6 +797,18 @@
           <dc:Bounds x="3532" y="820" width="57" height="27" />
         </bpmndi:BPMNLabel>
       </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="S1373649849862_event_subprocess_compensate_start_interrupting" bpmnElement="event_subprocess_compensate_start_interrupting">
+        <dc:Bounds x="3542" y="775" width="36" height="36" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="3532" y="820" width="57" height="27" />
```

Comment:
New shapes overlap with the existing ones when displaying the diagram

Reply:
I don't understand...

Reply:
When displaying the content of this file, we currently cannot see the added events, because they have the same bounds (aka position and dimensions) as existing elements that are loaded after the new ones. So the new elements are hidden by the existing ones on the html page.
I don't think this is something we have documented or you have talked about with @csouchet.

This part of the tests is currently a pain and we haven't already worked on that topic (this is a remaining task of #433)

Reply:
I see. 
My first impression was 'this file is too big, it's a pain to edit it' 
-> maybe if you manage to split this overlap issue won't be anymore, and it will be really easier to work on it.

Reply:
I agree to manage myself the position of the shapes in another PR, until we find a way to make it easier 😉

Reply:
This will be managed in #678 or in a new Pull Request